configs/agent/ci.toml

```diff
@@ -55,6 +55,16 @@ reserved-cpu = 1
 reserved-mem = "1G"
 reserved-disk = "8G"
 
+[logging.graylog]
+host = "localhost"
+port = 12201
+
+# If True, validate the Graylog server’s certificate. In this case specifying ca_certs is also required.
+validate = false
+
+ca_certs = ""
```

Comment:
Please keep it "kebab-case" instead of "snake_case" to be consistent with existing configs.

Reply:
Also are these certificates for the client authentication or server authentication (like usual HTTPS)?

Reply:
ca_certs is used for server authentication. It's should be a PEM-encoded CA certificate file which is used to verify the server's TLS certificate, there is more details here: https://go2docs.graylog.org/5-0/setting_up_graylog/https.html?tocpath=Setting%20up%20Graylog%7CSecuring%20Graylog%7C_____2